ios/FastImage/FFFastImageCacheNoParamMapper.m

```diff
@@ -0,0 +1,61 @@
+//
+//  FFFastImageCacheNoParamMapper.m
+//  FastImage
+//
+//  Created by Kamil Badyla on 06/03/2019.
+//  Copyright © 2019 vovkasm. All rights reserved.
+//
+
+#import "FFFastImageCacheNoParamMapper.h"
+#import <SDWebImage/SDWebImageManager.h>
+
+@implementation NSURL (StaticUrl)
+
+- (NSURL*)staticURL {
+	return [[NSURL alloc] initWithScheme:self.scheme host:self.host path:self.path];
+}
+
+@end
+
+@interface FFFastImageCacheNoParamMapper ()
+
+@property (strong) NSMutableSet *staticUrls;
+
+@end
+
+@implementation FFFastImageCacheNoParamMapper
+
++ (instancetype)shared {
+	static FFFastImageCacheNoParamMapper *_shared = nil;
+	static dispatch_once_t onceToken;
+	dispatch_once(&onceToken, ^{
+		_shared = [FFFastImageCacheNoParamMapper new];
+	});
+	
+	return _shared;
+}
+
+- (id)init {
+	self = [super init];
+	if (self) {
+		_staticUrls = [NSMutableSet new];
+		[[SDWebImageManager sharedManager] setCacheKeyFilter:^NSString * _Nullable(NSURL * _Nullable url) {
```

Comment:
to correct the error   `Sending 'NSString * _Nullable (^)(NSURL * _Nullable __strong)' to parameter of incompatible type 'id<SDWebImageCacheKeyFilter> _Nullable'`
```suggestion
		[SDWebImageCacheKeyFilter cacheKeyFilterWithBlock:^NSString * _Nullable(NSURL * _Nullable url) {
```